x/evm/keeper/attest.go

```diff
@@ -203,37 +204,37 @@ func (k Keeper) attestRouter(ctx sdk.Context, q consensus.Queuer, msg consensust
 			logger.Debug("TX proof attestation")
 			tx, err := winner.GetTX()
 			if err != nil {
-				logger.WithError(err).Error("Failed to get TX")
+				logger.With(err).Error("Failed to get TX")
 				return err
 			}
 			if k.isTxProcessed(ctx, tx) {
 				// somebody submitted the old transaction that was already processed?
 				// punish those validators!!
-				logger.WithError(err).Error("TX already processed")
+				logger.With(err).Error("TX already processed")
```

Comment:
This is not resolved yet. Please revert these changes

Reply:
sure